src/Sulu/Bundle/SecurityBundle/Admin/SuluSecurityContactsContentNavigationProvider.php

```diff
@@ -2,30 +2,36 @@
 
 namespace Sulu\Bundle\SecurityBundle\Admin;
 
-use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationInterface;
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationProviderInterface;
 use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationItem;
 use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
 
-class SuluSecurityContactsContentNavigation implements ContentNavigationInterface
+class SuluSecurityContactsContentNavigationProvider implements ContentNavigationProviderInterface
```

Comment:
no this prefixes are pretty useless ... the question for me is: is this content-navigation the right term for this?